pom.xml

```diff
@@ -440,7 +440,7 @@
     <swagger-parser-version>1.0.6-SNAPSHOT</swagger-parser-version>
     <scala-version>2.10.4</scala-version>
     <felix-version>2.3.4</felix-version>
-    <swagger-core-version>1.5.1-M2</swagger-core-version>
+    <swagger-core-version>1.5.2-M2-SNAPSHOT</swagger-core-version>
```

Comment:
@tomtit There is a Java generator that uses examples but the code cannot be merged back to the official repo as the generated code is in Java 8 but swagger-codegen has to be built in Java 7 container. 
Here is the repo
https://github.com/networknt/swagger-codegen
And here is the petstore generated
https://github.com/networknt/light-java-example/tree/master/petstore